package.json

```diff
@@ -77,7 +77,7 @@
     "@types/chai": "^4.0.0",
     "@types/chai-as-promised": "^7.1.0",
     "@types/firebase-token-generator": "^2.0.28",
-    "@types/jsonwebtoken": "^8.5.0",
+    "@types/jsonwebtoken": "^8.5.5",
```

Comment:
I'm wondering if we should just downgrade and pin this dependency to v8.5.1. Implementing all these workarounds to deal with a badly versioned dependency doesn't sound very good to me. WDYT?

Reply:
Agreed. Created a new PR to pin the dependency. I will close this for now.